src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -156,7 +156,7 @@ public static FeedbackSessionsForm getFormForNewFs(FeedbackSessionAttributes fee
                                                                               ? Const.INT_UNINITIALIZED  
                                                                               : feedbackSession.gracePeriod);
         
-        newFsForm.isSubmitButtonDisabled= courseIds.isEmpty();
+        newFsForm.isSubmitButtonDisabled = courseIdOptions.get(0).getContent().equals("No active courses!");
```

Comment:
Is there no better way than a brittle string comparison?
How about ensuring `courseIds` is indeed empty when there are no active courses or even shifting part of the logic for having no active courses into this method from `InstructorFeedbacksPageData`?

What exactly is an "active course"?
Does the instructor need permission to modify its session for it to be active for that instructor?
What should be the value of `isShowNoCoursesMessage` here in this case?